/rectangle_controller:
  ros__parameters:
    cropping:
        x_width: 0.3                                    # width of the specimen in x direction (forward-backward) in meters
        y_width: 1.525                                    # width of the specimen in y direction (left-right) in meters
        theta: 0.0                                      # rotation around z-axis as radians
        x_off: 1.0                                      # offset in x direction (forward-backward) in meters
        y_off: -0.025                                   # offset in y direction (left-right) in meters
        z_min: -0.15                                    # lower limit for height in meters
        z_max: 0.75                                     # upper limit for height in meters